osu.Game.Rulesets.Osu/Edit/DrawableOsuEditorRuleset.cs

```diff
@@ -46,15 +47,23 @@ public OsuEditorPlayfield()
                 HitPolicy = new AnyOrderHitPolicy();
             }
 
+            private readonly WeakList<DrawableHitObject> drawableHitObjects = new WeakList<DrawableHitObject>();
+
             [BackgroundDependencyLoader]
             private void load(OsuConfigManager config)
             {
                 hitAnimations = config.GetBindable<bool>(OsuSetting.EditorHitAnimations);
+                hitAnimations.BindValueChanged(_ =>
+                {
+                    foreach (var d in drawableHitObjects) d.RefreshStateTransforms();
```

Comment:
The only tiny note I have is that this could be replaced with `HitObjectContainer.Objects/AliveObjects` if I'm reading it right.

Reply:
That will be much better, yeah.